packages/cli/tests/unit/rpc/explain/pattern.spec.ts

```diff
@@ -0,0 +1,25 @@
+describe('Regex patterns', () => {
+  const EXCLUDE_DOT_APPMAP_DIR = /(^|[/\\])\.appmap([/\\]|$)/;
+  const EXCLUDE_DOT_NAVIE_DIR = /(^|[/\\])\.navie([/\\]|$)/;
```

Comment:
Ideally these would be replaced by the [exports in `collectContext.ts`](https://github.com/getappmap/appmap-js/blob/9f1f6ca12255e20ffd0397ffa17e352d95d1bfde/packages/cli/src/rpc/explain/collectContext.ts#L11-L13)

Reply:
Sigh.